uploadArchives fails in Travis for releases

Issue related to #239 

The nexus plugin failed multiple times uploading full staged repositories.
https://travis-ci.org/osmlab/atlas/builds/440233428

```
> Task :uploadArchives
Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.8.1/userguide/command_line_interface.html#sec:command_line_warnings
BUILD SUCCESSFUL in 1m 44s
11 actionable tasks: 1 executed, 10 up-to-date
Promote repository
> Task :closeRepository
Requested operation wasn't successful in first try. Retrying maximum 20 times with 2 seconds delay between.
Attempt 21/21 failed. WrongNumberOfRepositories was thrown with message 'Wrong number of received repositories in state 'open'. Expected 1, received 8'. Giving up. Configure longer timeout if necessary.
> Task :closeRepository FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':closeRepository'.
> Wrong number of received repositories in state 'open'. Expected 1, received 8
```

It seems that `gradle uploadArchives`, when running from a Travis VM (in GCE I believe) creates 7 or 8 partial repositories, instead of only one with all the artifacts.

I tried uploading archives from laptop and only one staging repository showed up.

Screenshots of the staging repositories:

<img width="1120" alt="image" src="https://user-images.githubusercontent.com/1944860/46897990-dd3d3080-ce3a-11e8-869c-17c6e4be2fbc.png">

<img width="364" alt="image" src="https://user-images.githubusercontent.com/1944860/46897995-eaf2b600-ce3a-11e8-92b3-9823c2a0397e.png">

<img width="398" alt="image" src="https://user-images.githubusercontent.com/1944860/46898015-08278480-ce3b-11e8-9956-ea484e22183a.png">

--- a/.travis.yml
+++ b/.travis.yml
@@ -25,6 +25,8 @@ branches:
 language: java
 jdk:
   - oraclejdk8
+sudo: false
+dist: trusty
 before_install:
   - chmod -R ug+x .travis
   - .travis/install.sh

